lib/compress/zstd_compress.c

```diff
@@ -288,6 +288,15 @@ static int ZSTD_resolveExternalSequenceValidation(int mode) {
 #endif
 }
 
+/* Sets the default maximum block size. */
+static size_t ZSTD_resolveMaxBlockSize(size_t maxBlockSize) {
```

Comment:
I presume you mean "defaultMaxBlockSize" ?

Reply:
I actually intended to follow the naming scheme of the previous "resolution" methods for the cctx parameters –– essentially just "resolving" the `maxBlockSize` value to the default if no value was present.

I think it's reasonable to keep the naming scheme for consistency, but the method annotation is definitely inaccurate. "Resolve maxBlockSize to the default if no value is present" might be more accurate?

Reply:
You're right, consistency matters, and I missed it was part of a larger list of `ZSTD_resolve*()` symbols.

OK for the updated annotation.